pkg/agent/flowexporter/connections/conntrack.go

```diff
@@ -44,13 +44,13 @@ func filterAntreaConns(conns []*flowexporter.Connection, nodeConfig *config.Node
 		srcIP := conn.TupleOrig.SourceAddress
 		dstIP := conn.TupleReply.SourceAddress
 
-		// Only get Pod-to-Pod flows.
+		// Consider Pod-to-Pod, Pod-To-Service and Pod-To-External flows.
```

Comment:
strange, I thought we had fixed that comment in an earlier PR

Reply:
Yes but the comments and log changes were not properly carried over when IPv6 branch is merged with master.